### PR TITLE
update my extension exstone

### PR DESCRIPTION
**Pull request by qimo601**
extract the light density parameters of urinary calculi

### CODE REVIEW

File: exStone.s4ext [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# list dependencies
+# - These should be names of other modules that have .s4ext files
+# - The dependencies will be built first
+depends     NA
+
+# Inner build directory (default is ".")
+build_subdirectory .
+
+# homepage
+homepage    http://slicer.org/slicerWiki/index.php/Documentation/Nightly/Extensions/exStone
```
Comment (fedorov) [inline exStone.s4ext]: this page has no content

File: exStone.s4ext [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+# Firstname1 Lastname1 ([SubOrg1, ]Org1), Firstname2 Lastname2 ([SubOrg2, ]Org2)
+# For example: Jane Roe (Superware), John Doe (Lab1, Nowhere), Joe Bloggs (Noware)
+contributors LiuZhaoBang (Sibet), John Doe (AnyWare Corp.)
```
Comment (fedorov) [inline exStone.s4ext]: Did you forget to remove Joe Doe? Is your name LiuZhaoBang or you forgot to include spaces?

File: exStone.s4ext [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+# Match category in the xml description of the module (where it shows up in Modules menu)
+category    Examples
+
+# url to icon (png, size 128x128 pixels)
+iconurl     http://www.example.com/Slicer/Extensions/exStone.png
```
Comment (fedorov) [inline exStone.s4ext]: this URL is invalid

File: exStone.s4ext [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+# Give people an idea what to expect from this code
+#  - Is it just a test or something you stand behind?
+status
```
Comment (fedorov) [inline exStone.s4ext]: what is the status?

File: exStone.s4ext [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+# url to icon (png, size 128x128 pixels)
+iconurl     http://www.example.com/Slicer/Extensions/exStone.png
+
+# Give people an idea what to expect from this code
+#  - Is it just a test or something you stand behind?
+status      
+
+# One line stating what the module does
+description This is an example of a simple extension
```
Comment (fedorov) [inline exStone.s4ext]: can you give a meaningful description of your extension?

File: exStone.s4ext [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+Urinary calculi analysis module
+
+# Space separated list of urls
+screenshoturls http://www.example.com/Slicer/Extensions/exStone/Screenshots/1.png
```
Comment (fedorov) [inline exStone.s4ext]: invalid

File: exStone.s4ext (hunk 1 above)
Comment (qimo601) [inline exStone.s4ext]: I'm sorry. I have just finished developing the extension code. Now ,I want to test how to publish the extension.

I was doing this work   completely in accordance with this web page  instructions.
link: http://slicer-devel.65872.n3.nabble.com/the-extension-dll-can-t-be-loaded-correctly-in-official-software-td4037500.html


Can you give me some advice on how to publish this extension?

File: exStone.s4ext (hunk 3 above)
Comment (qimo601) [inline exStone.s4ext]: Should I choose a stone image to replace this default file？Then push to the github.
image path:C:\workSpace\exStone\exStone.png

File: exStone.s4ext (hunk 5 above)
Comment (qimo601) [inline exStone.s4ext]: I will update the  description in the exStone.s4ext

File: exStone.s4ext (hunk 4 above)
Comment (qimo601) [inline exStone.s4ext]: i will update these information.   This file is generated by the Extension Wizard. I did not add too much information.

File: exStone.s4ext (hunk 1 above)
Comment (fedorov) [inline exStone.s4ext]: Sure. 

Please follow the instructions here: http://wiki.slicer.org/slicerWiki/index.php/Documentation/Nightly/Developers/Tutorials/BuildTestPackageDistributeExtensions

If you read step 8, it says: "Create a documentation page for your extension."

**Pull request closed without merging**